core/src/main/scala/cats/std/string.scala

```diff
@@ -1,7 +1,24 @@
 package cats
 package std
 
-trait StringInstances extends algebra.std.StringInstances {
+trait StringInstances {
+  implicit val stringOrder: Order[String] = new StringOrder
+  implicit val stringMonoid = new StringMonoid
   implicit val stringShow: Show[String] =
     Show.fromToString[String]
 }
+
+class StringOrder extends Order[String] {
```

Comment:
Just one observation on your comment: the class isn't part of the kernel, so won't be subject to quite the same strictures as the kernel types.